fplll/defs.h

```diff
@@ -286,6 +286,8 @@ enum PrunerFlags
   PRUNER_NELDER_MEAD      = 0x8,
   // Verbosity
   PRUNER_VERBOSE = 0x10,
+  // Optimize w.r.t to all coefficients (by default it optimizes w.r.t half)
+  PRUNER_OPTIMIZE_FULL = 0x20
```

Comment:
Maybe just `PRUNER_FULL` since `OPTIMIZE` is kinda redundant?

Reply:
Yep, did the succinct one.